Plugin/PaymentVaultDeleteToken.php

```diff
@@ -12,11 +12,13 @@
 namespace Adyen\Payment\Plugin;
 
 use Adyen\AdyenException;
+use Adyen\Model\Recurring\DisableRequest;
 use Adyen\Client;
 use Adyen\Payment\Helper\Data;
 use Adyen\Payment\Helper\Requests;
 use Adyen\Payment\Helper\Vault;
 use Adyen\Payment\Logger\AdyenLogger;
+use Adyen\Service\RecurringApi;
```

Comment:
`RecurringApi` was added to imports but service `recurringService` is still using the old implementation.